.github/ISSUE_TEMPLATE/bug report.yml

```diff
@@ -49,4 +49,18 @@ body:
     id: context
     attributes:
       label: Additional context
-      placeholder: Add the link to the page(s) or any other context/screenshots related to the bug report here.
\ No newline at end of file
+      placeholder: Add the link to the page(s) or any other context/screenshots related to the bug report here.
+on:
```

Comment:
You're specifying a GitHub action inside a GitHub issue template. This will not work.
The only thing you need to assign is either the `projects: [...]` tag (see more: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) or adding an automatic filter for the project itself (see more: https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/adding-items-automatically).